recipes/jarvis-tools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "jarvis-tools" %}
+{% set version = "2020.10.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jarvis-tools-{{ version }}.tar.gz
+  sha256: b4cc45267100fe2cc4d114d1453ad39e87d35244ce511c44b1fc99ca277f8a07
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib-base
+    - numpy
+    - pytest
+    - python
+    - requests
+    - scipy
+    - spglib
+    - toolz
+    - xmltodict
+
+test:
+  imports:
+    - jarvis
+    - jarvis.ai
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://jarvis.nist.gov
+  summary: 'jarvis-tools: an open-source software package for data-driven atomistic materials design'
+  license: NIST-PD
+  license_file: LICENSE
```

Comment:
Did you by any chance create an upstream issue that the license should be packaged in their tarball?

Reply:
Done, https://github.com/usnistgov/jarvis/issues/128